code/game/objects/items/food/soup.dm

```diff
@@ -23,6 +23,7 @@
 		desc = "A wish come true!"
 		reagents.add_reagent(/datum/reagent/consumable/nutriment, 9)
 		reagents.add_reagent(/datum/reagent/consumable/nutriment/vitamin, 1)
+		trash_type = /obj/item/reagent_containers/cup/bowl
```

Comment:
why do we only spawn the bowl as trash if it's got nutriment in it? can you not eat it otherwise?